scripts/completion/dvc.bash

```diff
@@ -24,26 +24,26 @@ _dvc_destroy='-f --force'
 _dvc_diff='-t --show-json --show-hash'
 _dvc_fetch='-j --jobs -r --remote -a --all-branches -T --all-tags -d --with-deps -R --recursive'
 _dvc_fetch_COMPGEN=_dvc_compgen_DVCFiles
-_dvc_get_url=''
+_dvc_gc='-a --all-branches -T --all-tags -w --workspace -c --cloud -r --remote -f --force -p --projects -j --jobs'
```

Comment:
`--all-commits` was hidden before. But, #3429 made it visible.

Reply:
Oh I see that issue now. It is not deprecated, just hidden.

Reply:
#2681

Reply:
@casperdcl, `--all-commits` is now visible and supported. Please check #3429.

Reply:
Ah I see; didn't realise as it's not in the latest release. Right.